lib/9pfs/9pfs.h

```diff
@@ -39,63 +39,150 @@
 
 #include <vfscore/prex.h>
 
-/*
- * Currently supports only the 9P2000.u variant of the protocol.
+/**
+ * enum uk_9pfs_proto - Protocol version;
+ * currently only the 9P2000.u variant of the protocol is supported
  */
 enum uk_9pfs_proto {
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Unix systems
+	 */
 	UK_9P_PROTO_2000U,
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Linux systems
+	 */
 	UK_9P_PROTO_2000L,
+	/*
+	 * The number of existent protocols;
+	 * in this case, two (UK_9P_PROTO_2000U and UK_9P_PROTO_2000L)
```

Comment:
```suggestion
	 * in this case, two (`UK_9P_PROTO_2000U` and `UK_9P_PROTO_2000L`)
```
Add things that are not english words betwen backticks, so a spellcheck would not complain. Do this for the whole file.